public/themes/lara-dark-cyan/theme.css

```diff
@@ -776,7 +776,7 @@
     background: #111827;
     width: 22px;
     height: 22px;
-    color: rgba(255, 255, 255, 0.87);
+    color: #212529;
```

Comment:
Yes the styles are here: https://github.com/primefaces/primereact

Reply:
you would make a change to the SCSS and regenerate all the themes.

Reply:
Instead of changing the theme color, I configured the checkboxIcon property to function similarly to the DataTable's checkbox property. :) ([b1b99f4](https://github.com/primefaces/primereact/pull/7406/commits/b1b99f45f9c86f5e6e4be413b04a6f7ad2536064))


 https://github.com/primefaces/primereact/blob/e86ddccdb70cfd93110b33523dfb1eddc376a361/components/lib/datatable/RowCheckbox.js#L35-L40